### DISCUSSION TITLE
Value of Transfac Pro database

### DISCUSSION THREAD

**Discussion by dburkhardt** [aertslab/pycistarget#21] [Q&A]
I'm trying to understand the value of the proprietary transfac database. I don't see much information about how much more information is gained using Transfac Pro. This is about 30% of your motif collection, but it's not clear how much additional value that brings to building the GRN. Can you shed any light on that?

**Comment by SeppeDeWinter** [comment]
Hi @dburkhardt 

Please see figure S3, pannel g and h in the SCENIC+ manuscript: https://www.biorxiv.org/content/10.1101/2022.08.19.504505v1.
In pannel h (p) denotes the public collection (i.e. without transfac pro).

Here we did this comparison and noticed that the differences are minimal.

Best,

Seppe

**Reply by dburkhardt** [reply]
This is great, thank you! I was wondering what the (a), (u), (c), and (p) variants of pycistopic represented there. It isn't in the figure legend that I can see on BioRxiv, unless I'm missing something. What are they?

**Reply by SeppeDeWinter** [reply]
Indeed, that was missing from the legend

a: uses a single (archetype) motif per cluster, instead of using all motifs per cluster
u: unclustered motif collection
c: clustered motif collection, using all motifs per cluster
